### ISSUE TITLE
NullPointerException when specifying charge carrier mass

### ISSUE THREAD

**Issue by ahoiching** [MSGFPlus/msgfplus#103]
**Describe the bug**
Here's the machine, system and error message when I was running:
MS-GF+ Release (v2020.03.14) (14 March 2020)
Java 11.0.1 (Oracle Corporation)
Mac OS X (x86_64, version 10.15.5)
java.lang.NullPointerException
	at edu.ucsd.msjava.params.DoubleParameter.parse(DoubleParameter.java:21)
	at edu.ucsd.msjava.msdbsearch.SearchParams.parseConfigParamFile(SearchParams.java:489)
	at edu.ucsd.msjava.msdbsearch.SearchParams.parse(SearchParams.java:239)
	at edu.ucsd.msjava.ui.MSGFPlus.runMSGFPlus(MSGFPlus.java:74)
	at edu.ucsd.msjava.ui.MSGFPlus.main(MSGFPlus.java:58)

**How the error message would be gone**
If I comment out the "ChargeCarrierMass=1.00727649" in the configuration file, the error message is gone and could be run through (I have not finished running through one sample yet)...Using the -ccm handle in the command line would return to the same error.

**Comment by alchemistmatt** [comment]
Bug confirmed.  

Command line and stack trace when using `-ccm`
```
C:\DMS_Programs\Java\jre8\bin\java.exe  -Xmx4000M -jar C:\DMS_Programs\MSGFPlus.jar -s QC_Mam_19_01.mzML -o QC_Mam_19_01_msgfplus.mzid -d C:\DMS_Temp_Org\ID_004225_C1CEE570.fasta -ccm 1.00727649 -conf MSGFPlus_Tryp_MetOx_StatCysAlk_20ppmParTol.txt

Exception in thread "main" java.lang.NullPointerException
        at edu.ucsd.msjava.params.DoubleParameter.parse(DoubleParameter.java:21)
        at edu.ucsd.msjava.params.ParamManager.parseParams(ParamManager.java:371)
        at edu.ucsd.msjava.ui.MSGFPlus.main(MSGFPlus.java:45)
```
Command line and stack trace when using `ChargeCarrierMass=1.00727649` in the parameter file:
```
C:\DMS_Programs\Java\jre8\bin\java.exe  -Xmx4000M -jar C:\DMS_Programs\MSGFPlus.jar -s QC_Mam_19.mzML -o QC_Mam_19_01_msgfplus.mzid -d C:\DMS_Temp_Org\ID_004225_C1CEE570.fasta -conf MSGFPlus_Tryp_MetOx_StatCysAlk_20ppmParTol.txt

MS-GF+ Release (v2020.03.14) (14 March 2020)
Java 1.8.0_232 (AdoptOpenJDK)
Windows 10 (amd64, version 10.0)
java.lang.NullPointerException
        at edu.ucsd.msjava.params.DoubleParameter.parse(DoubleParameter.java:21)
        at edu.ucsd.msjava.msdbsearch.SearchParams.parseConfigParamFile(SearchParams.java:489)
        at edu.ucsd.msjava.msdbsearch.SearchParams.parse(SearchParams.java:239)
        at edu.ucsd.msjava.ui.MSGFPlus.runMSGFPlus(MSGFPlus.java:74)
        at edu.ucsd.msjava.ui.MSGFPlus.main(MSGFPlus.java:58)
```
I will fix this soon.

**Comment by alchemistmatt** [comment]
Fixed via commit c5fa52dde2a3128050f3c1675969b0f0c919c49b
See release 2020.06.16 at https://github.com/MSGFPlus/msgfplus/releases/

**Comment by alchemistmatt** [comment]
I noticed that `FloatParameter.java` had similar code as DoubleParameter.java and thus ran the risk of a NPE; fixed via commit 7657886b24b83aa0c126028fe60eed9f9c0c964b
I re-generated [Release 2020.06.16](https://github.com/MSGFPlus/msgfplus/releases/tag/v2020.06.16)
